crates/autopilot/src/run_loop.rs

```diff
@@ -334,7 +324,7 @@ impl RunLoop {
         let current = self.current_block.borrow().number;
         let start = current.saturating_sub(start_offset);
         let deadline = current.saturating_add(max_wait_time_blocks);
-        tracing::debug!(%current, %start, %deadline, ?tag, "waiting for tag");
+        tracing::debug!(%current, %start, %deadline, ?id, %submission_address, "waiting for settlement");
```

Comment:
nit:

```suggestion
        tracing::debug!(%current, %start, %deadline, ?id, ?submission_address, "waiting for settlement");
```

The `ethereum_types::H160` display implementation does not print the whole address.